cmd/vela-worker/operate.go

```diff
@@ -23,6 +26,35 @@ func (w *Worker) operate() error {
 	if err != nil {
 		return err
 	}
+	// Define the database representation of the worker
+	// and register itself in the database
+	registryWorker := new(library.Worker)
+	registryWorker.SetHostname(w.Config.Hostname)
+	registryWorker.SetAddress(w.Config.Server.Address)
+	registryWorker.SetRoutes(w.Config.Queue.Routes)
+	registryWorker.SetActive(true)
+	registryWorker.SetLastCheckedIn(time.Now().UTC().Unix())
+	err = w.register(registryWorker)
+	if err != nil {
+		logrus.Error(err)
+	}
+
+	// spawn goroutine for phoning home
+	go func() {
+		for {
+			// sleep for the configured time
+			time.Sleep(w.Config.CheckIn)
+
+			// set checking time to now and call the server
+			registryWorker.SetLastCheckedIn(time.Now().UTC().Unix())
+			_, _, err := w.VelaClient.Worker.Update(registryWorker.GetHostname(), registryWorker)
+
+			// if unable to update the worker, log the error but allow the worker to continue running
+			if err != nil {
+				logrus.Errorf("unable to update worker %s in database: %v", registryWorker.GetHostname(), err)
```

Comment:
I believe the error message for this line and [this other line](https://github.com/go-vela/worker/blob/worker_table/cmd/vela-worker/register.go#L42) are still different.

Should they be?